alws/build_planner.py

```diff
@@ -91,6 +91,16 @@ def load_platform_flavors(self, flavors):
         if db_flavors:
             self._platform_flavors = db_flavors
 
+    def is_beta_build(self):
+        if not self._platform_flavors:
+            return False
+        found = False
+        for flavor in self._platform_flavors:
+            if bool(re.search(r'(-beta)$', flavor.name, re.IGNORECASE)):
+                found = True
+                break
```

Comment:
```
strings = ['123', '456', '789', 'aaa']
for string in strings:
    if bool(found := re.search('aaa', string)):
        break
        return found
 ```

Reply:
I'm still getting used to this syntax, will do the change

Reply:
I agree.

Reply:
This comment for conversation above)